lib/android/app/src/main/java/com/reactnativenavigation/anim/TopbarCollapsingBehavior.java

```diff
@@ -0,0 +1,81 @@
+package com.reactnativenavigation.anim;
+
+
+import android.view.View;
+
+import com.facebook.react.uimanager.events.EventDispatcher;
+import com.reactnativenavigation.interfaces.ScrollEventListener;
+import com.reactnativenavigation.utils.UiThread;
+import com.reactnativenavigation.views.TopBar;
+
+public class TopbarCollapsingBehavior {
```

Comment:
Change tense to present.
`TopBarCollapsingBehavior` -> `TopBarCollapseBehavior`